pkgs/top-level/all-packages.nix

```diff
@@ -4185,6 +4185,10 @@ in
     openssl = openssl_1_1;
   };
 
+  # Update this when adding the newest nodejs major version!
+  nodejs-latest = nodejs-12_x;
+  nodejs-slim-latest = nodejs-slim-12_x;
```

Comment:
I suggest using `nodejs_latest` and `nodejs-slim_latest`. `_` is usually used to denote that the version follows, which in this case is the "latest" one. It's also `linuxPackages_latest`.